src/main/java/uk/gov/hmcts/reform/preapi/media/MediaKind.java

```diff
@@ -202,7 +223,7 @@ private void startLiveEvent(String liveEventName) {
     private void stopAndDeleteLiveEvent(String liveEventName) {
         try {
             mediaKindClient.stopLiveEvent(liveEventName);
-        } catch (NotFoundException e) {
+        } catch (FeignException.NotFound e) {
             throw new NotFoundException("Live Event: " + liveEventName);
         }
```

Comment:
Its not possible to get `FeignException.NotFound` anymore because of `FeignErrorDecoder`. 

```suggestion
        } catch (NotFoundException e) {
            return;
        }
```